### DISCUSSION TITLE
Is Direct Pay IRA benefit modeled in the REopt tool?

### DISCUSSION THREAD

**Discussion by kmsunflower** [NREL/REopt-API-Analysis#145] [5 - Financial Input Questions] [answered]
Hello, I am assessing microgrid development for a nonprofit entity, so no tax. I want to account for the IRA direct pay provisions allowing the entity to benefit from direct pay incentive, but wanted to know if this is included in the REopt tool. If so, how is best to account for it?

**Comment by adfarth** [comment, marked as answer]
You can set the owner's tax rate to 0%, and set the PV or Wind Federal "Incentive based on percentage of cost (%)" (in the code, `federal_itc_fraction`) to the ITC value (by default these are set to 30%). For a battery system, the web tool input is "Total percentage-based incentive (%)" (`total_itc_fraction` in the code) and the default is currently 30% as well.
